plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -879,6 +879,11 @@ class FlingGestureTest(
     assertEquals(expectedFlingDuration, mapAnimationOptionsSlot.captured.duration)
   }
 
+  @Test
```

Comment:
Add / update test for `ROTATION_ANGLE_THRESHOLD` constant as well?

Reply:
it's not in the scope of this pr, but yeah I can add it. :)